for/joomla.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: Codeception for Joomla
+hero: joomla_hero.html
+sidebar: |
+
+  ## Codeception Tests
+
+  * Combine **all testing levels** (acceptance, functional, unit)
+  * **Scenario-Driven**:  described in easy to get PHP DSL
+  * Provide common actions and assertions
+
+  ## Reference
+
+  * [Joomla Browser](http://codeception.com/addons#joomla-browserhttpsgithubcomjoomla-projectsjoomla-browser) 
+  * [Demo Tested Extension](https://github.com/joomla-extensions/weblinks#tests)
+
+---
+
+## Setup
+
+With Codeception you can create [Unit](http://codeception.com/docs/05-UnitTests), [Functional](http://codeception.com/docs/04-FunctionalTests) and [Api](http://codeception.com/docs/10-WebServices) tests, but the present documentation will covers only how to start with  [Acceptance](http://codeception.com/docs/03-AcceptanceTests) tests in Joomla and Joomla Extensions in order to  check the UI and work of a system as a whole. 
```

Comment:
I think there is no need for introduction to unit/functional/api tests as they are not related to Joomla.